editor.planx.uk/src/@planx/components/PlanningConstraints/Editor.tsx

```diff
@@ -33,34 +43,117 @@ function PlanningConstraintsComponent(props: Props) {
+              <Input
+                format="large"
+                name="title"
+                placeholder={formik.values.title}
+                value={formik.values.title}
+                onChange={formik.handleChange}
+              />
+            </InputRow>
+            <InputRow>
+              <RichTextInput
+                name="description"
+                placeholder="Description"
+                value={formik.values.description}
+                onChange={formik.handleChange}
+              />
+            </InputRow>
             <InputRow>
               <Input
                 format="data"
                 name="fn"
                 placeholder={formik.values.fn}
                 value={formik.values.fn}
-                onChange={formik.handleChange}
+                disabled
               />
             </InputRow>
           </InputGroup>
+          <InputGroup label="Which constraints do you want to check?">
+            <TableContainer>
+              <Table size="small">
+                <TableHead>
+                  <TableRow>
+                    <TableCell>
+                      <Checkbox
+                        color="primary"
+                        checked={true}
+                        disabled={true}
+                        inputProps={{
+                          "aria-label": "select all constraints",
+                        }}
+                      />
+                    </TableCell>
+                    <TableCell sx={{ fontWeight: FONT_WEIGHT_BOLD }}>
+                      Constraints
+                    </TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {availableDatasets
+                    .sort((a, b) => a.val.localeCompare(b.val))
+                    .map((dataset, i: number) => (
+                      <TableRow key={i}>
+                        <TableCell sx={{ verticalAlign: "top" }}>
+                          <Checkbox
+                            color="primary"
+                            checked={true}
+                            disabled={true}
+                          />
+                        </TableCell>
+                        <TableCell>
+                          <InputGroup>
+                            <InputRow>
+                              <Input
+                                format="large"
+                                name="text"
+                                value={dataset.text}
+                                disabled
+                              />
+                            </InputRow>
+                            <InputRow>
+                              <Input
+                                format="data"
+                                name="val"
+                                value={dataset.val}
+                                disabled
+                              />
+                            </InputRow>
+                          </InputGroup>
```

Comment:
I'm now using disabled `InputRows` within each `TableRow`/`TableCell` here, as I think this has a more similar feel to Question & Checklist "Options" ? I think a table, rather than list, is still right here as it will facilitate the "(De)Select all" action across rows. 

These styles can absolutely be further tweaked before we wire-up & actually allow interaction!  

Here's my original draft using plain `Typography`, which I think looked more crowded? 
![Screenshot from 2024-05-24 10-08-15](https://github.com/theopensystemslab/planx-new/assets/5132349/01ea80c5-54b0-4b33-ad94-9a90d44817f2)